app/components/browse/prefix_selector.html.erb

```diff
@@ -1,12 +1,31 @@
-<nav class="alpha-filter">
-  <ol class="pagination pagination-sm justify-content-center">
-    <li class="page-item <%= 'active' if prefix.blank? %>">
-      <%= link_to 'All', clear_prefix_path, class: 'page-link' %>
-    </li>
-    <% ('A'..'Z').to_a.each do |letter| %>
-      <li class="page-item <%= 'active' if prefix == letter %>">
-        <%= link_to letter, prefix_path(letter), class: 'page-link' %>
-      </li>
+<div class="row my-3">
+
+  <div class="col-9">
+    <nav class="alpha-filter">
+      <ol class="pagination pagination-sm mb-0">
+        <li class="page-item <%= 'active' if prefix.blank? %>">
+          <%= link_to t('.all'), clear_prefix_path, class: 'page-link' %>
+        </li>
+        <% ('A'..'Z').to_a.each do |letter| %>
+          <li class="page-item <%= 'active' if prefix == letter %>">
+            <%= link_to letter, prefix_path(letter), class: 'page-link' %>
+          </li>
+        <% end %>
+      </ol>
+    </nav>
+  </div>
+
+  <div class="col-3">
+    <%= form_tag form_url, method: :get, id: 'browse-form', role: 'browse' do %>
+      <label for="prefix" class="sr-only"><%= t('.label') %></label>
+      <div class="input-group">
+        <span class="input-group-text"><%= t('.label') %></span>
```

Comment:
Wrap this in `<span class="input-group-prepend">` for a little nicer rendering:

### Before
<img width="295" alt="Screen Shot 2021-09-01 at 4 39 05 PM" src="https://user-images.githubusercontent.com/639920/131741745-1a2a0128-8212-40e0-a718-339a61559629.png">

### After
<img width="300" alt="Screen Shot 2021-09-01 at 4 38 48 PM" src="https://user-images.githubusercontent.com/639920/131741769-36006f25-ee70-494c-a746-86f4e64caec7.png">

Reply:
Thanks, I've added that in. I'm not sold on any of the UI stuff yet, this is just to get something up there to play with.